wgsl/index.bs

```diff
@@ -6238,6 +6252,9 @@ the following exceptions:
     * Other operations are required to preserve denormalized numbers.
 * The accuracy of operations is given in [[#floating-point-accuracy]].
 
+[SHORTNAME] attemps to reduce the scope of applied exceptions to floating-point operations
+in functions with [=attribute/precise_math=] attributes.
```

Comment:
Might be worth a backreference to the attribute, at the definition of 'reassociation'